trunk/source/Library-Automata/src/de/uni_freiburg/informatik/ultimate/automata/nestedword/operations/IsEmpty.java

```diff
@@ -345,6 +382,7 @@ private DoubleDecker<STATE> dequeue() {
 		// If available, take a state pair that has been discovered by taking an internal or a return transition or a
 		// summary. If not, take a state pair that has been discovered by taking a call transition.
 		case DFS -> dequeueGivenQueues(mQueue, mQueueCall);
+		default -> throw new IllegalArgumentException("Unexpected value: " + mStrategy);

```

Comment:
Please avoid adding a `default` case to this previously-exhaustive switch.

Ideally, `IsEmpty` should not need special handling for its subclass `IsEmptyParallel`. Do you really need the additional enum value? (it's also a bit strange since it couples a search mode (BFS) with the fact that the search is parallelized -- can they be split?)

If there is no way to avoid having the additional enum value, use `case PARALLEL ->` here.